jsonrpc/filter_manager.go

```diff
@@ -17,7 +16,7 @@ import (
 )
 
 var (
-	ErrFilterDoesNotExists              = errors.New("filter does not exists")
```

Comment:
I noticed this typo, and also have proposed returning a string containing the words `filter not found`.

This is a condition that clients need to be able to distinguish from other errors, and the Ethereum JSON/RPC does not have a formal standard for this. However, other Ethereum clients by convention contain this string. So to reduce complexities for developers integrating with multiple clients, I suggest following that convention.

Reply:
Agree thank you